### ISSUE TITLE
Conditional Logic

### ISSUE THREAD

**Issue by IronRunes** [the-engine-room/alidade#25]
Some slides need to be loaded conditionally, depending on previous answers or paths hte user has taken. This needs to be implemented. (cfr. slides no. 1.3, 2.5->2.6, Step 4)

**Comment by IronRunes** [comment]
- Show user different set of questions in 1.3 depending on whether they click 'inside your organisation' or 'outside your organisation'
- depending on checkboxes in 4.1 different set of questions is displayed in 4.2
- Display slide 2.6 only if user selects 'we will build a new tool' in 2.5.
- Only show step 4 if user selects 'We need help' option in step 3. recap

**Comment by IronRunes** [comment]
- [x] Different set of questions on 1.3
- [x] Depending on checkbox selection 4.1, different questions on 4.2
- [x] Slide 2.6 appears only upon "New Tool" commitment on 2.5
- [x] Step 4 is **displayed** only if user checks the "We need help" option on Step 3 Recap

**Comment by IronRunes** [comment]
hey @thomwithoutanh i was looking at the step 3 recap, and I can't seem to find a "We need help" option. Is it to be intended that it's the "New Tool" option?

**Comment by thomwithoutanh** [comment]
@IronRunes Yes - this is an error in the mockups. It should be the slide that comes *after* the Step 3 recap (currently called 'time to make a decision' - it should be 'Do you need help from a technical partner?'). 

I'll fix the rest of the content later. 

But the buttons should be: 
- We need to find a technical partner 
- We can complete this work on our own

Does that make sense?

**Comment by thomwithoutanh** [comment]
Content updated, by the way.
